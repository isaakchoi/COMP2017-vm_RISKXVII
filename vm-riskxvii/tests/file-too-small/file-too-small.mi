Filler data
Filler data
Filler data
Filler data
Filler data
Filler data
Filler data
Filler data
Filler data
Filler data
Filler data
Filler data
Filler data
Filler data
Filler data
Filler data
Filler data
Filler data
Filler data
Filler data
Filler data
Filler data
Filler data
Filler data
Filler data
Filler data
Filler data
Filler data
Filler data
Filler data
Filler data
Filler data
Filler data
Filler data
Filler data
Filler data
Filler data
Filler data
Filler data
Filler data
