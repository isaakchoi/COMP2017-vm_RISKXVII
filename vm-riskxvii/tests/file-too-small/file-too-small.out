ERR: Invalid file input size: [480 bytes]
